runtime/doc/debugger.txt

```diff
@@ -114,7 +114,7 @@ other settings apply.
 
 The debugger features were added for use with Sun's Visual WorkShop Integrated
 Programming Environment (ipe).  However, they were done in as generic a manner
```

Comment:
Is "generic a manner" a mistake of "a generic manner"?
(Not fixed in this patch.)

Reply:
No, I believe it is correct in the context of the rest of the sentence. If you search online for "generic a manner", you will find several instances of "as generic a manner as possible".